packages/identity-service/test/lib/app.js

```diff
@@ -25,7 +25,23 @@ async function getApp () {
   }
   const audiusLibs = { captcha, solanaWeb3Manager }
 
+  const fpClient = {
+    getVisitorHistory: async (visitorId) => {
+      return {
+        visitorId,
+        visits: [{
+          confidence: { score: 0.8 },
```

Comment:
saw we used 0.7 in code. can you provide a bit of context on how we choose these values (maybe as a comment)